website/source/docs/agent/configuration/vault.html.md

```diff
@@ -51,20 +51,20 @@ vault {
 - `task_token_ttl` `(string: "")` - Specifies the TTL of created tokens when
```

Comment:
https://github.com/hashicorp/nomad/pull/1931/files#diff-4c8f84ccf0c97f1b28502dd7644a7f48R42

Update the default